.github/util/re-version.sh

```diff
@@ -75,7 +75,9 @@ do
   (cd $workdir/finalize-jar && jar cfm $workdir/$jar $workdir/tmp-manifest.mf .)
 
   cp $workdir/$jar $outdir
-  rename.ul 0-SNAPSHOT $version $outdir/$jar
+  RENAME_SNAPSHOTS=$(ls $outdir/$jar | sed -e "s/0-SNAPSHOT/$version/g")
```

Comment:
@sayaliM0412 there's no chance the version will ever have a `/` in it, right?  I've been bitten by using that particular delimiter with sed before.

Reply:
hmm, it's replacing `0-SNAPSHOT` with `$version` globally. 
🤔

Reply:
I believe we will never have a version like 4.21/0 unless it's a typo - so I believe it's fine.